motoman_driver/MotoPlus/SimpleMessage.h

```diff
@@ -72,6 +72,7 @@ typedef enum
 
 	ROS_MSG_MOTO_JOINT_TRAJ_PT_FULL_EX = 2016,
 	ROS_MSG_MOTO_JOINT_FEEDBACK_EX = 2017,
+	ROS_MSG_MOTO_SELECT_TOOL = 2018,

```

Comment:
This should be accompanied by an update to [REP-I4](https://github.com/ros-industrial/rep/blob/master/rep-I0004.rst).

Reply:
Ok

Reply:
Links: https://github.com/ros-industrial/rep/pull/26.

Reply:
REP update was merged.